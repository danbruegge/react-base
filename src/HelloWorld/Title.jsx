// @flow
import styled from 'styled-components';

export default styled.h1`
    background-color: SeaGreen;
    color: #fff;
    font-size: 40px;
    margin-bottom: 5px;
    padding-left: 5px;
`;
