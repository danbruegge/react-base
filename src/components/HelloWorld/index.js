import React from 'react';

import { Title, Message } from 'app/components/core';

const HelloWorld = () => (
    <div>
        <Title>Hello World</Title>
        <Message>This is a hello world message.</Message>
    </div>
);

export default HelloWorld;
