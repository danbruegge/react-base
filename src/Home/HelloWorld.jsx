import React from 'react';

import Title from './Title';
import Message from './Message';

export default () => (
    <div>
        <Title>Hello World</Title>
        <Message>This is a hello world message.</Message>
    </div>
);
