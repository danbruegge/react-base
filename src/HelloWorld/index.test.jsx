// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import HelloWorld from './index';

test('Render HelloWorld correctly', () => {
    const component = <HelloWorld />;
    const tree = renderer.create(component).toJSON();

    expect(tree).toMatchSnapshot();
});
