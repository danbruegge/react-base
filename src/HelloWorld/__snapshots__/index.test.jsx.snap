// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HelloWorld correctly 1`] = `
.c0 {
  background-color: SeaGreen;
  color: #fff;
  font-size: 40px;
  margin-bottom: 5px;
  padding-left: 5px;
}

.c1 {
  color: DimGrey;
  font-size: 25px;
  margin: 0;
  padding: 5px;
}

<div>
  <h1
    className="c0"
  >
    Hello World
  </h1>
  <p
    className="c1"
  >
    This is a hello world message.
  </p>
</div>
`;
